yb-voyager/cmd/importData.go

```diff
@@ -1060,7 +1064,10 @@ func executeSqlStmtWithRetries(conn **pgx.Conn, sqlInfo sqlInfo, objType string)
 			}
 
 			// DROP INDEX in case INVALID index got created
-			dropIdx(*conn, fullyQualifiedObjName)
+			err2 := dropIdx(*conn, fullyQualifiedObjName)
+			if err2 != nil {
+				utils.ErrExit("%v: %v", err, err2)
```

Comment:
return err2 ?

Reply:
`err` variable is already used in retry logic, don't want to override that.